tests/unit/util/test_token_manager.py

```diff
@@ -3,9 +3,10 @@
 
 import pytest
 
-from praw.util.token_manager import BaseTokenManager, FileTokenManager
```

Comment:
Can you leave a test which ensures trying to use BaseTokenManager directly raises an exception?